packages/shared/lib/core/app/enums/platform-option.enum.ts

```diff
@@ -1,4 +1,5 @@
 export enum PlatformOption {
+    iOS = 'ios',
```

Comment:
```suggestion
    IOS = 'ios',
```
based on our coding conventions, all enums need to be pascal case